---
layout: page
title: Resume
permalink: /resume/
icon: far fa-address-card
type: page
---

* content
{:toc}


Zijun Zhang
============

<hr />
<div style="width: 700px;margin: 10 auto;padding: 0;overflow: auto;float: left;">
	<div style="float:left;">         
		<p>Bioinformatics IDP, UCLA</p>
		<p>Los Angeles, CA90095</p>
		<p>USA</p>
	</div>
	<div style="float:right;">
		<p><a href="mailto:zj.z@ucla.edu" target="blank">zj.z@ucla.edu</a></p>
		<p><a href="https://zj-zhang.github.io/" target="blank">https://zj-zhang.github.io/</a></p>
		<p><a href="https://www.linkedin.com/in/zijun-zhang-7859a07a/" target="blank">LinkedIn</a></p>
	</div>
</div>



Education
---------

2014-2019 (expected)
:   Ph.D. in Bioinformatics; UCLA (Los Angeles, USA)


2010-2014
:   B.Sc. in Bioinformatics; Zhejiang University (Hangzhou, China)


Publications
-------------
* <strong>Zhang, Zijun</strong>; Park, Eddie; Lin, Lan; Xing, Yi; <a>A panoramic view of RNA modifications: exploring new frontiers</a> (Genome Biology, 2018)
* Park, Eddie; Pan, Zhicheng; <strong>Zhang, Zijun</strong>; Lin, Lan; Xing, Yi; <a>The expanding landscape of alternative splicing variation in human populations</a> (American Journal of Human Genetics, 2018)
* <strong>Zhang, Zijun</strong>; Xing, Yi; <a>CLIP-seq analysis of multi-mapped reads discovers novel functional RNA regulatory sites in the human transcriptome</a> (Nucleic acids research, 2017)
* Liu, Lili; <strong>Zhang, Zijun</strong>; Sheng, Taotao; Chen, Ming; <a>DEF: an automated dead-end filling approach based on quasi-endosymbiosis</a> (Bioinformatics, 2017)
* Liu, Lili; Mei, Qian; Yu, Zhenning; Sun, Tianhao; <strong>Zhang, Zijun</strong>; Chen, Ming; <a>An integrative bioinformatics framework for genome-scale multiple level network reconstruction of rice</a> (Journal of integrative bioinformatics,2013)
* Liu, Lili; <strong>Zhang, Zijun</strong>; Mei, Qian; Chen, Ming; <a>PSI: a comprehensive and integrative approach for accurate plant subcellular localization prediction</a> (PLoS One, 2013)

Projects
--------------------

**CLAM**: CLIP-seq Analysis of Multi-mapped reads
:   Open-source python package for calling peaks in CLIP/RIP-seq data.


**DARTS**: Deep learning augmented Analysis of RNA-seq Transcript Splicing 
:   TBD


Skills
----------------------------------------
* Programming languages: Python, R, scikit, Keras, Tensorflow, Theano, Matlab, AWS, HTML, CSS, PHP, Jekyll, SQL, C/C++, Unix/SGE, Git
* Statistics: Regularized regression, Bayesian statistics and Bayesian inference, Causal Inference, Hierarchical modeling, MCMC, multivariate analysis
* Machine/Deep learning: Supervised(CNN, LSTM, adaboost/xgboost, SVM, random forest), Unsupervised(clustering, autoencoder, factor analysis), Reinforcement(controller network)