---
layout: resume
title: Resume
permalink: /resume/
icon: far fa-address-card
type: page
---


<body>
<h1_title id="zijun-zhang">Zijun Zhang</h1_title>

<!--
<table border="0">
<tbody>
<tr class="odd">
<td>Bioinformatics IDP, UCLA</td>
<td style="text-align: right;"><a href="mailto:zj.z@ucla.edu">zj.z@ucla.edu</a></td>
</tr>
<tr class="even">
<td>Los Angeles, CA90095</td>
<td style="text-align: right;"><a href="https://zj-zhang.github.io/" class="uri" target="_blank">https://zj-zhang.github.io/</a></td>
</tr>
<tr class="odd">
<td>USA</td>
<td style="text-align: right;"></td>
</tr>
</tbody>
</table>
-->
<div style="width:100%;margin:20 auto;padding: 10;overflow: auto;float: left;background-color:white">
  <div style="float:left;">         
    <p>Simons Foundation</p>
    <p>New York, NY10010</p>
    <p>USA</p>
  </div>
  <div style="float:right;">
    <p><a href="mailto:zj.z@ucla.edu" target="blank">zj.z@ucla.edu</a></p>
    <p><a href="https://zj-zhang.github.io/" target="blank">https://zj-zhang.github.io/</a></p>
    <p><a href="https://www.linkedin.com/in/zijun-zhang-7859a07a/" target="blank">LinkedIn</a></p>
  </div>
</div>

<h2 id="education"><i class="fa fa-graduation-cap"></i> Education</h2>
<dl>

<dt>2019-Present</dt>
<dd>Postdoc Research Fellow; Simons Foundation (New York, USA)
</dd>

<dt>2014-2019</dt>
<dd>Ph.D. in Bioinformatics; UCLA (Los Angeles, USA)
</dd>
<dd>Thesis: <a href="https://escholarship.org/uc/item/11z9d3bm" target="_blank">Computational methods to elucidate post-transcriptional gene regulation using high-throughput sequencing data</a></dd>

<dt>2015-2019</dt>
<dd>M.S. in Statistics:; UCLA (Los Angeles, USA)
</dd>
<dd>Thesis: <a href="https://escholarship.org/uc/item/6dr8f6kj" target="_blank">Neural Architecture Search for Biological Sequences</a></dd>

<dt>2010-2014</dt>
<dd>B.Sc., Zhejiang University (Hangzhou, China)
</dd>
</dl>
<h2 id="publications"><i class="fas fa-file-alt"></i> Publications</h2>
<ul>

<li><strong>Zijun Zhang</strong>, Zhicheng Pan, Yi Ying, Zhijie Xie, Samir Adhikari, John Phillips, Russ P Carstens, Douglas L Black, Yingnian Wu, Yi Xing; <a>Deep-learning augmented RNA-seq analysis of transcript splicing</a> (Nature Methods, 2019)</li>

<li><strong>Zijun Zhang</strong>, Linqi Zhou, Liangke Gou, Yingnian Wu; <a>Neural Architecture Search for Joint Optimization of Predictive Power and Biological Knowledge</a> (arXiv, 2019)</li>

<li><strong>Zhang, Zijun</strong>; Park, Eddie; Lin, Lan; Xing, Yi; <a>A panoramic view of RNA modifications: exploring new frontiers</a> (Genome Biology, 2018)</li>

<li>Park, Eddie; Pan, Zhicheng; <strong>Zhang, Zijun</strong>; Lin, Lan; Xing, Yi; <a>The expanding landscape of alternative splicing variation in human populations</a> (American Journal of Human Genetics, 2018)</li>

<li><strong>Zhang, Zijun</strong>; Xing, Yi; <a>CLIP-seq analysis of multi-mapped reads discovers novel functional RNA regulatory sites in the human transcriptome</a> (Nucleic acids research, 2017)</li>

<li>Liu, Lili; <strong>Zhang, Zijun</strong>; Sheng, Taotao; Chen, Ming; <a>DEF: an automated dead-end filling approach based on quasi-endosymbiosis</a> (Bioinformatics, 2017)</li>

<li>Liu, Lili; Mei, Qian; Yu, Zhenning; Sun, Tianhao; <strong>Zhang, Zijun</strong>; Chen, Ming; <a>An integrative bioinformatics framework for genome-scale multiple level network reconstruction of rice</a> (Journal of integrative bioinformatics,2013)</li>

<li>Liu, Lili; <strong>Zhang, Zijun</strong>; Mei, Qian; Chen, Ming; <a>PSI: a comprehensive and integrative approach for accurate plant subcellular localization prediction</a> (PLoS One, 2013)</li>

</ul>
<h2 id="projects"><i class="fab fa-github"></i> Projects</h2>
<dl>

<dt><strong>BioNAS</strong></dt>
<dd><em>Neural architecture search for Bioinformatics</em>
</dd>
<dd>AutoML framework for designing powerful and interpretable neural networks.
</dd>
<dd><a href="https://github.com/zj-zhang/BioNAS-pub" target=_blank>https://github.com/zj-zhang/BioNAS-pub</a>
</dd>

<dt><strong>CLAM</strong></dt>
<dd><em>CLIP-seq Analysis of Multi-mapped reads</em>
</dd>
<dd>Open-source python package for calling peaks in CLIP/RIP-seq data.
</dd>
<dd><a href="https://github.com/Xinglab/CLAM" target=_blank>https://github.com/Xinglab/CLAM</a>
</dd>

<dt><strong>DARTS</strong></dt>
<dd><em>Deep learning augmented Analysis of RNA-seq Transcript Splicing</em>
</dd>
<dd>Bayesian inference utilizing deep-learning prediction informative prior
</dd>
<dd><a href="https://github.com/Xinglab/DARTS" target=_blank>https://github.com/Xinglab/DARTS</a></dd>
</dl>
<h2 id="skills"><i class="fas fa-cogs"></i> Skills</h2>
<ul>
<li>Programming languages: Python, R, scikit, Keras, Tensorflow, Theano, Matlab, AWS, HTML, CSS, PHP, Jekyll, SQL, C/C++, Unix/SGE, Git</li>
<li>Statistics: Regularized regression, Bayesian statistics and Bayesian inference, Causal Inference, Hierarchical modeling, MCMC, multivariate analysis</li>
<li>Machine/Deep learning: Supervised(CNN, LSTM, adaboost/xgboost, SVM, random forest), Unsupervised(clustering, autoencoder, factor analysis), Reinforcement(controller network)</li>
</ul>
</body>
